webrtc.html

```diff
@@ -7677,6 +7677,11 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
                   to <var>option</var>'s <code>priority</code> member.</p>
                 </li>
                 <li>
+                  <p>If <a>[[\Negotiated]]</a> is <code>true</code> and
+                  <a>[[\DataChannelId]]</a> is <code>null</code>, <a>throw</a>
+                  a <code>TypeError</code>.</p>
+                </li>
+                <li>
```

Comment:
nit: Doesn't really make a difference, but for organizational purposes it would make sense to move this up one step, so it's right after `[[DataChannelId]]` being set.

Reply:
Fixed.